website/members/templates/members/email/yearly_study_check.html

```diff
@@ -0,0 +1,12 @@
+{% extends "email/html_email.html" %}
+
+{% block content %}
+  Dear {{ name }},<br>
+
+  <p>You're a study member of Thalia. This means that you'll be a member as long as you're enrolled in either a Computing Science or an Information Sciences programme.</p>
+
+  <p>Are you still studying? Then please follow the link in this URL <a href="{{ renewal_url }}">{{ renewal_url }}</a><br>
+  Did you graduate or stop with your studies for any other reason? Then you can simply ignore this message!</p>
```

Comment:
I think these mails could use a little nore explanation: why do you follow the link? What happens of you ignore it?